src/poliastro/tests/test_bodies.py

```diff
@@ -69,3 +88,348 @@ def test_from_relative():
+        value_from_poliastro = [i.value for i in Venus.rot_elements_at_epoch(epoch)]
+
+        body_code = spice.bodn2c("Venus")
+
+        BODY_POLE_RA = spice.gdpool("BODY" + str(body_code) + "_POLE_RA", 0, 3)
+        BODY_POLE_DEC = spice.gdpool("BODY" + str(body_code) + "_POLE_DEC", 0, 3)
+        BODY_PM = spice.gdpool("BODY" + str(body_code) + "_PM", 0, 3)
+
+        body_RA = BODY_POLE_RA[0] + BODY_POLE_RA[1] * T
+        body_DEC = BODY_POLE_DEC[0] + BODY_POLE_DEC[1] * T
+        body_PM = BODY_PM[0] + BODY_PM[1] * d
+        value_from_spice = [body_RA, body_DEC, body_PM]
+
+        for i, j in zip(value_from_poliastro, value_from_spice):
+            assert_quantity_allclose(i, j)
+
+    def test_Earth(self, set_epoch):
+        epoch, T, d = set_epoch
+        value_from_poliastro = [i.value for i in Earth.rot_elements_at_epoch(epoch)]
+
+        body_code = spice.bodn2c("Earth")
+
+        BODY_POLE_RA = spice.gdpool("BODY" + str(body_code) + "_POLE_RA", 0, 3)
+        BODY_POLE_DEC = spice.gdpool("BODY" + str(body_code) + "_POLE_DEC", 0, 3)
+        BODY_PM = spice.gdpool("BODY" + str(body_code) + "_PM", 0, 3)
+
+        body_RA = BODY_POLE_RA[0] + BODY_POLE_RA[1] * T
+        body_DEC = BODY_POLE_DEC[0] + BODY_POLE_DEC[1] * T
+        body_PM = BODY_PM[0] + BODY_PM[1] * d
+        value_from_spice = [body_RA, body_DEC, body_PM]
+
+        for i, j in zip(value_from_poliastro, value_from_spice):
+            assert_quantity_allclose(i, j)
+
+    def test_Mars(self, set_epoch):
+        epoch, T, d = set_epoch
+        value_from_poliastro = [i.value for i in Mars.rot_elements_at_epoch(epoch)]
+
+        body_code = spice.bodn2c("Mars")
+
+        BODY_POLE_RA = spice.gdpool("BODY" + str(body_code) + "_POLE_RA", 0, 3)
+        BODY_POLE_DEC = spice.gdpool("BODY" + str(body_code) + "_POLE_DEC", 0, 3)
+        BODY_PM = spice.gdpool("BODY" + str(body_code) + "_PM", 0, 3)
+
+        body_RA = BODY_POLE_RA[0] + BODY_POLE_RA[1] * T
+        body_DEC = BODY_POLE_DEC[0] + BODY_POLE_DEC[1] * T
+        body_PM = BODY_PM[0] + BODY_PM[1] * d
+        value_from_spice = [body_RA, body_DEC, body_PM]
+
+        for i, j in zip(value_from_poliastro, value_from_spice):
+            assert_quantity_allclose(i, j)
+
+    def test_Jupiter(self, set_epoch):
+        epoch, T, d = set_epoch
+        value_from_poliastro = [i.value for i in Jupiter.rot_elements_at_epoch(epoch)]
+
+        body_code = spice.bodn2c("Jupiter")
+
+        BODY_POLE_RA = spice.gdpool("BODY" + str(body_code) + "_POLE_RA", 0, 3)
+        BODY_POLE_DEC = spice.gdpool("BODY" + str(body_code) + "_POLE_DEC", 0, 3)
+        BODY_PM = spice.gdpool("BODY" + str(body_code) + "_PM", 0, 3)
+        BODY5_NUT_PREC_ANGLES = spice.gdpool("BODY5_NUT_PREC_ANGLES", 0, 30)
+        BODY599_NUT_PREC_RA = spice.gdpool("BODY599_NUT_PREC_RA", 0, 15)
+        BODY599_NUT_PREC_DEC = spice.gdpool("BODY599_NUT_PREC_DEC", 0, 15)
+
+        Ja = BODY5_NUT_PREC_ANGLES[20] + BODY5_NUT_PREC_ANGLES[21] * T
```

Comment:
I was shocked when I saw the code the first time some days ago, but then I realized it's exactly the same logic we're applying in `bodies.py`. Perhaps it would be better to have functions that can be reused, so they just receive appropriate numerical values? That would make these tests shorter.

Reply:
I'll start working on the functions.

Reply:
I tried to write functions that would help in reducing the length of the tests. The problem is that the big tests are unique and there isn't a generalized way of calculating them. So, I'll just end up passing all the coefficients in the functions, which won't really make the tests shorter Instead, I was thinking I could just write them in a more compact like:
![new_format](https://user-images.githubusercontent.com/35529737/55317285-d01a0980-548d-11e9-9451-bd5deb63f40a.png)
instead of
![old](https://user-images.githubusercontent.com/35529737/55317297-d5775400-548d-11e9-80e9-c800c80213a2.png)